src/Microsoft.Extensions.OData.Migration/Formatters/ODataMigrationMessageWrapper.cs

```diff
@@ -0,0 +1,231 @@
+﻿// ------------------------------------------------------------------------------
+// <copyright company="Microsoft Corporation">
+//     Copyright © Microsoft Corporation. All rights reserved.
+// </copyright>
+// ------------------------------------------------------------------------------
+
+namespace Microsoft.AspNet.OData.Formatter
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Linq;
+    using System.Text.RegularExpressions;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.OData;
+
+    /// <summary>
+    /// Copy of ODataMessageWrapper for IODataRequestMessage and IODataResponseMessage.
```

Comment:
If it's copied code. Explain why it has to be copied and add a reference to where it is copied from.